### PR TITLE
fix(DB/Quest): improved No Fly Zone

### PR DESCRIPTION
**Pull request by grimgravy**
Quest: No Fly Zone  available through the website [here](https://www.wowhead.com/quest=12815/no-fly-zone)

## Changes Proposed:
-  Fix: Raidus 3yards
-  Fix: removed the Blood Presence aura. Correct function added smart_scripts Cast (11)
-  Fix: Gravity and Hover
-  Fix: Plague Strike ability of the npc Bone Gryphon

## Tests Performed:

-  was performed windows 7 64bits. Not a single error was found and it is Functional

## How to Test the Changes:

1. quest add 12815
2. tele icecrown
3. go to the zone indicated by the map and test

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1625841824336043500.sql [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+-- Radius 5 Yards
+-- I tested it with Knock Back (350), which is a very far distance. Therefore, I reduced it to 200
+-- https://www.wowhead.com/spell=54469/plague-strike
+UPDATE `spell_dbc` SET `EffectRadiusIndex_1`='8', `EffectRadiusIndex_2`='8', `EffectRadiusIndex_3`='8' WHERE  `ID`=54469;
+UPDATE `spell_dbc` SET `EffectMiscValue_3` = 200 WHERE (`ID` = 54469);
```
Comment (FrancescoBorzi) [inline data/sql/updates/pending_db_world/rev_1625841824336043500.sql:30]:
```suggestion
UPDATE `spell_dbc` SET `EffectRadiusIndex_1`=8, `EffectRadiusIndex_2`=8, `EffectRadiusIndex_3`=8, `EffectMiscValue_3` = 200 WHERE  `ID`=54469;
```

**Comment by FrancescoBorzi** [comment]
@pklloveyou can you please test this?

**Comment by pklloveyou** [comment]
ok

**Comment by pklloveyou** [comment]
Tested

**Review by FrancescoBorzi — APPROVED** [review]

**Comment by FrancescoBorzi** [comment]
good job @grimgravy and thanks @pklloveyou for testing this!

**Pull request merged**